<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Datenbank_Editor</class>
 <widget class="QMainWindow" name="Datenbank_Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datenbank_Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnQuit">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>640</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Beenden</string>
    </property>
    <property name="icon">
     <iconset resource="resources/icons/icons.qrc">
      <normaloff>:/16x16/16x16/quit.ico</normaloff>:/16x16/16x16/quit.ico</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>251</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Länder auswahl:</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>221</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioLand1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLand2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLand3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLand4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioLand5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>251</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Vereins auswahl:</string>
    </property>
    <widget class="QListView" name="listView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>221</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioVerein1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein6">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein7">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein9">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein10">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein11">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein12">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein13">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein14">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein15">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein16">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein17">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein18">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein19">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVerein20">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Vereins Daten:</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>92</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="labelVerein1">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelVerein2">
        <property name="text">
         <string>Liga zugehörigkeit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelVerein3">
        <property name="text">
         <string>Bankguthaben:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelVerein4">
        <property name="text">
         <string>Stadt:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>141</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEditVereinName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditVereinLiga"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditVereinGeld"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditVereinStadt"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>351</width>
      <height>671</height>
     </rect>
    </property>
    <property name="title">
     <string>Spieler auswahl:</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>201</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox3_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Länder Daten:</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="labelLand1">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLand2">
        <property name="text">
         <string>Einwohner</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>161</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLineEdit" name="lineEditLandName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLandEinwohner"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="BtnSave">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>580</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
    <property name="icon">
     <iconset resource="resources/icons/icons.qrc">
      <normaloff>:/16x16/16x16/save.ico</normaloff>:/16x16/16x16/save.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnOpen">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>540</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Öffnen</string>
    </property>
    <property name="icon">
     <iconset resource="resources/icons/icons.qrc">
      <normaloff>:/16x16/16x16/open.ico</normaloff>:/16x16/16x16/open.ico</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Liga auswahl</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QComboBox" name="comboBoxLiga"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Liga Daten:</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="labelLiga1">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLineEdit" name="lineEditLigaName"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_ffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="separator"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info?</string>
    </property>
    <addaction name="action_ber"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/16x16/16x16/save.ico</normaloff>:/16x16/16x16/save.ico</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="action_ffnen">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/16x16/16x16/open.ico</normaloff>:/16x16/16x16/open.ico</iconset>
   </property>
   <property name="text">
    <string>Öffnen...</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="icon">
    <iconset resource="resources/icons/icons.qrc">
     <normaloff>:/16x16/16x16/quit.ico</normaloff>:/16x16/16x16/quit.ico</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="action_ber">
   <property name="text">
    <string>Über...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BtnQuit</sender>
   <signal>clicked(bool)</signal>
   <receiver>Datenbank_Editor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1225</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBeenden</sender>
   <signal>triggered(bool)</signal>
   <receiver>Datenbank_Editor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
